src/applications/static-pages/sass/static-pages.scss

```diff
@@ -23,41 +23,6 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 @import "modules/m-operating-status";
 @import "modules/m-facility-events";
 
-//=============================
-// Home page only styles.
-//=============================
-
-.popular-container {
```

Comment:
Weird, what were these for? I don't remember ever seeing these classes 🤔

Reply:
No idea, some of this stuff is 5+ years old